achievements/helpingHand.achievement.js

```diff
@@ -0,0 +1,38 @@
+var _ = require('lodash');
+
+var helpingHand = {
+  name: 'Helping Hand',
+  check: function(pullRequest, shall) {
+    var committedReviewers = reviewersWhoCommittedToPullRequest(pullRequest);
+
+    if (!_.isEmpty(committedReviewers)) {
+
+      var achievement = {
+        avatar: 'images/achievements/helpingHand.achievement.jpg',
+        name: 'Helping Hand',
+        short: 'Let me help you finishing up here...',
+        description: 'You\'ve committed to a Pull Request you are reviewing',
+        relatedPullRequest: pullRequest._id
+      };
+
+      _.forEach(committedReviewers, function(reviewer) {
+        shall.grant(reviewer.username, achievement);
+      });
+    }
+  }
+};
+
+
+function reviewersWhoCommittedToPullRequest(pullRequest) {
+  var committedReviewers = [];
+
+  _.forEach(pullRequest.commits, function(commit) {
+    if (_.includes(pullRequest.reviewers, commit.creator.username)) {
```

Comment:
Also, `_.includes` won't work here since it checks if a value is included in a collection.

I think `_.find` can be better used here like so:
```
if( _.find(pullRequest.reviewers, {username: commit.author.username}) ) {}
```